.github/workflows/ci.yml

```
@@ -0,0 +1,83 @@
+
+name: Server
+
+on:
+  push:
+    branches:
+      - version-13-hotfix
+      - version-13
+    paths-ignore:
+      - '**.js'
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.js'
+      - '**.md'
+
+concurrency:
+  group: version-13-landa-${{ github.event.number }}
+  cancel-in-progress: true
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    name: Server
+
+    services:
+      mariadb:
+        image: mariadb:10.3
```

```suggestion
        image: mariadb:10.5
```

We don't have 10.3 running in prod anymore